src/input/output_event/mod.rs

```diff
@@ -12,6 +14,7 @@ pub enum OutputEvent {
     Evdev(InputEvent),
     Uinput(UinputOutputEvent),
     DualSense(SetStatePackedOutputData),
+    XpadUhid(XpadUhidOutputData),
```

Comment:
Do we need this output event? We would only need this if we plan on emulating an XpadUhid target device that needs to send output events to a source device.

Reply:
Uhid xpad devices require we send rumble data back to the source device as a URB packet.

Reply:
But do you have plans on making a Uhid Xpad target device? If not, then we shouldn't need this.

Reply:
All Bluetooth Xbox controllers need this. Currently only the xpadneo DKMS driver works for them, but it is out of tree and incompatible with xpad. I've been talking to Luke Jones about an xpad_uhid driver but that isn't certain and this could be a solution in the meantime.